src/components/InstanceNameHandler.js

```diff
@@ -194,7 +194,7 @@ function getRandomFromList(list) {
 }
 // eslint-disable-next-line
 function getRandomInstanceName() {
-  return getRandomFromList(adjectives) + getRandomFromList(nouns);
+  return getRandomFromList(adjectives) + ' ' + getRandomFromList(nouns);
```

Comment:
Why add a space in the random name?

Reply:
I just thought it looked better as proper english. No other reason really